### PR TITLE
Update Home Page

### PR DESCRIPTION
**Pull request by isharailanga**
This PR is to update the home page as follows:

- Update intro paragapraph formatting
- Remove the space allocated for the youtube intro video (as there's no Securirty dedicated Youbube intro)
- Rearrange and add a new tile to report vulnerabilities
- Move "Report Vulnerability" section to a different page

### CODE REVIEW

**Comment by isharailanga** [comment]
screencapture-127-0-0-1-8000-2022-11-09-10_23_35

File: en/docs/index.md [hunk 1]
```diff
@@ -7,14 +7,18 @@ template: templates/single-column.html
...
         <h1>Welcome to the WSO2 Security and Compliance Documentation!</h1>
     </header>
-    <div class="md-main .md-content" style="float:left; width: 45%;  text-align:justify; max-height:100%; ">
-        <p style="margin-bottom:10%">
-        Security is our top priority and we proactively look for ways to improve the security of our products, services, and infrastructure. Our security program is transparent so you can be confident when using our products and services, and infrastructure for mission-critical projects.      
+    <div class="md-main .md-content" style="float:center; width: 45%;  text-align:justify; max-height:100%; ">
+        <p>
+        Security is our top priority and we proactively look for ways to improve the security of our products, services,
+        and infrastructure. Our security program is transparent so you can be confident when using our products and 
+        services, and infrastructure for mission-critical projects.
```
Comment (nilminiwso2) [inline en/docs/index.md:14]:
```suggestion
        services and infrastructure for mission-critical projects. 
```

File: en/docs/index.md [hunk 2]
```diff
@@ -26,8 +30,7 @@ template: templates/single-column.html
                     </div>
                     <div class="card-content">
                         <p class="title">Security Processes and Programs</p>
-                        <a href="http://www.google.com"></a>
-                        <p class="hint">Read our security processes and programs to understand how we manage various aspects of security in our products.</p>
+                        <p class="hint">How we manage various aspects of security in WSO2 products</p>
```
Comment (nilminiwso2) [inline en/docs/index.md:33]:
```suggestion
                        <p class="hint">Learn how we manage various aspects of security in WSO2 products</p>
```

File: en/docs/index.md [hunk 3]
```diff
@@ -40,8 +43,8 @@ template: templates/single-column.html
                         <i class="material-icons md-36">library_books</i>
                     </div>
                     <div class="card-content">
-                        <p class="title">Security Guideliness</p>
-                        <p class="hint">Read our guidelines to understand how to enforce security-first development and deployment practices.</p>
+                        <p class="title">Security Guidelines</p>
+                        <p class="hint">How to enforce security-first development and deployment practices</p>
```
Comment (nilminiwso2) [inline en/docs/index.md:47]:
```suggestion
                        <p class="hint">Learn how to enforce security-first development and deployment practices</p>
```

File: en/docs/report-vulnerabilities.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# Report a Security Issue
+
+We welcome all contributions from our user community, developers, customers and security researchers to reinforce our
```
Comment (nilminiwso2) [inline en/docs/report-vulnerabilities.md:3]:
```suggestion
We welcome all contributions from our user community, developers, customers, and security researchers to reinforce our 
```

File: en/docs/report-vulnerabilities.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+# Report a Security Issue
+
+We welcome all contributions from our user community, developers, customers and security researchers to reinforce our 
+security. We strongly encourage you to report security issues to our private and highly confidential security mailing
```
Comment (nilminiwso2) [inline en/docs/report-vulnerabilities.md:4]:
```suggestion
security of our products and services. We strongly encourage you to report security issues to our private and highly confidential security mailing 
```

File: en/mkdocs.yml [hunk 6]
```diff
@@ -277,6 +277,8 @@ nav:
       - '2020':
         - '': 'security-announcements/incident-clarifications/2020/index.md'
         - 'SolarWinds SUNBURST breach': 'security-announcements/incident-clarifications/2020/solarwinds-sunburst-breach.md'
+  - 'Report Vulnerabilities':
```
[the remark this answers is not in the document]
Comment (nilminiwso2) [reply en/mkdocs.yml:280]: name of the page seems to be `Report a Security Issue`. However, the menu is `Report Vulnerabilities`.
This looks a buggy presentation. We only have one page under `Report Vulnerabilities` (which will look odd in the left nav. Or have we removed the left nav from this section?).

However, I do agree that `Report Vulnerabilities` or `Report an Issue` deserves its own section. Currently, it is hidden under `Security Guidelines`. May be we can bring all content related to Vulnerabilities (including the guidelines) under this new menu.

The `Security Guidelines` should be reserved to `how to code securely` and `how to deploy securely`. Whereas, `how to report vulnerabilities` (guidelines) can go under the `Report Vulnerabilities` section. 

wdyt?

@ayomawdb

File: en/docs/index.md (hunk 1 above)
Comment (isharailanga) [reply en/docs/index.md]: Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8

File: en/docs/index.md (hunk 2 above)
Comment (isharailanga) [reply en/docs/index.md]: Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8

File: en/docs/index.md (hunk 3 above)
Comment (isharailanga) [reply en/docs/index.md]: Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8

File: en/docs/report-vulnerabilities.md (hunk 4 above)
Comment (isharailanga) [reply en/docs/report-vulnerabilities.md]: Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8

File: en/docs/report-vulnerabilities.md (hunk 5 above)
Comment (isharailanga) [reply en/docs/report-vulnerabilities.md]: Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8

File: en/docs/report-vulnerabilities.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
+# Report a Security Issue
+
+We welcome all contributions from our user community, developers, customers and security researchers to reinforce our 
+security. We strongly encourage you to report security issues to our private and highly confidential security mailing 
+lists mentioned below, before disclosing them in any forums, sites or other groups - public or private.
```
[the remark this answers is not in the document]
Comment (isharailanga) [reply en/docs/report-vulnerabilities.md]: Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8

**Comment by isharailanga** [comment]
Closing this PR as these changes are overriden with https://github.com/wso2/docs-security/pull/20/

**Pull request closed without merging**